sympy/matrices/dense.py

```diff
@@ -904,8 +907,11 @@ def col_del(self, i):
         col
         row_del
         """
-        for j in range(self.rows - 1, -1, -1):
-            del self._mat[i + j*self.cols]
```

Comment:
Oh. Here the original is an in-place update.
So it's probably okay to change to `del` in the above.
BUT... do a `git annotate` to check the history and see whether that's an accident or intentional.